### PR TITLE
Fix naming and directory structure

### PR DESCRIPTION
**Pull request by RyanMeulenkamp**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# yagl4j
+# frosty
```
Comment (thumbnail) [inline readme.md:1]: I just gave you (temporary) admin rights for this repo. Can you make sure the repo description and change the name.

File: src/native/cpp/library.h [hunk 2]
```diff
@@ -56,7 +56,7 @@ extern "C" {
  * game-pad manager is started and if no game-pad listener is hooked yet.
  *
  * @note This method binds to the
- *       com.copilot.yagl4j.GamePadManager.hookGamePadListener method.
+ *       com.copilot.frosty.GamePadManager.hookGamePadListener method.
```
Comment (thumbnail) [inline src/native/cpp/library.h]: wrong name

File: src/native/cpp/library.h [hunk 3]
```diff
@@ -65,7 +65,7 @@ extern "C" {
  */
 __attribute__ ((used))
 JNIEXPORT jboolean
-JNICALL Java_com_copilot_yagl4j_GamePadManager_hookGamePadListener(
+JNICALL Java_com_copilot_frosty_GamePadManager_hookGamePadListener(
```
Comment (thumbnail) [inline src/native/cpp/library.h]: wrong name

**Review by thumbnail — CHANGES_REQUESTED** [review]
Please update the c-code accordingly. Also i can't see how this compiled on your machine since the naming isn't synchronized between the c- and java-project. 

Also this PR contained alot of added `final` statements. which should've been in #2

**Review by thumbnail — DISMISSED** [review]

**Comment by thumbnail** [comment]
Since #10 hasn't been fixed yet I can't verify whether this PR is safe to merge; because `make` doesn't work locally. Could you describe the build steps?

**Comment by thumbnail** [comment]
I also noticed you use your company email for this repository; you can change that setting on a repository base using `git config user.email "ryan.meulenkamp@gmail.com"`.

**Comment by RyanMeulenkamp** [comment]
Steps to build:
```Shell 
cd ${PROJECT_DIR}/src/native/cpp
cmake . 
make
```

In Intellij, create a build configuration. Add the following vm parameter: `-Djava.library.path=/build`

**Comment by thumbnail** [comment]
Could you move 47643b3 to another PR so we can merge the rename PR separately?

**Comment by RyanMeulenkamp** [comment]
Done

File: readme.md [hunk 4]
```diff
@@ -14,6 +11,4 @@ The following packages are required:
 
 ## Contributors
 - Mark Lefering
-
-## License
```
Comment (romni) [inline readme.md:18]: no license for this project?

File: src/main/java/io/leonis/frosty/GamePadAxis.java [hunk 5]
```diff
@@ -24,8 +24,7 @@
    *
    * @param value The value of the controller axis.
    */
-  GamePadAxis(int value) {
-    // Setting value
+  GamePadAxis(final int value) {
```
Comment (romni) [inline src/main/java/io/leonis/frosty/GamePadAxis.java:27]: no explicit access modifier?

File: src/main/java/io/leonis/frosty/GamePadAxis.java [hunk 6]
```diff
@@ -36,14 +35,13 @@
    * @return The ControllerAxis.
    * @throws IllegalArgumentException -1 <= value <= 5.
    */
-  public static GamePadAxis fromValue(int value)
-      throws IllegalArgumentException {
-    if (value >= -1 && value <= 5) {
-      return values()[value + 1];
-    } else {
+  public static GamePadAxis fromValue(final int value) {
```
Comment (romni) [inline src/main/java/io/leonis/frosty/GamePadAxis.java:38]: is this secretly a constructor?

**Review by thumbnail — APPROVED** [review]
Catchy name. we might need a logo in the near future :)

File: src/main/java/io/leonis/frosty/GamePadAxis.java (hunk 5 above)
Comment (thumbnail) [reply src/main/java/io/leonis/frosty/GamePadAxis.java:27]: for an enum?

**Pull request merged**